📦 NEW: Node.js Community Page Content (#31)
nodejs/outreach: community.md

> # Content for NodeJS Community Webpage

Reviewer: Replace with Node.js

> # Content for Node.js Community Webpage

Reviewer 2: ```suggestion
# Node.js Community
```

## Introduction

> NodeJS is an open source software built, used, and maintained by the community. We believe community to be at the core of NodeJS' popularity. Amazing folks from different parts of the world, with a common interest, collaborate and shape this community.

Reviewer: Change to Node.js

> NodeJS is an open source software built, used, and maintained by the community. We believe community to be at the core of NodeJS' popularity. Amazing folks from different parts of the world, with a common interest, collaborate and shape this community.

Reviewer: Change NodeJS to Node.js

> Node.js is an open source software built, used, and maintained by the community. We believe community to be at the core of NodeJS' popularity. Amazing folks from different parts of the world, with a common interest, collaborate and shape this community.

Reviewer: `NodeJS' popularity,` — Node.js needs to changed

> Node.js is an open source software built, used, and maintained by the community. We believe community to be at the core of Node.js popularity. Amazing folks from different parts of the world, with a common interest, collaborate and shape this community.

Reviewer 2: ```suggestion
Node.js has one of the largest, most vibrant and innovative open source communities in the world. Node.js is built, used, and maintained by the community. We believe the community to be at the core of Node.js popularity. Amazing people from every part of the world, with a common interest, collaborate and shape this community.
```

> In its early days, when Node.js was still in its initial releases, it became extremely popular. Thousands of developers were using it. With the increasing usage, it became difficult for the Node.js core team to keep the software up to date with the user requirements. So thus formed the Node.js foundation in 2015.

Reviewer 2: This para is unnecessary, consider removing it.

> In its early days, when Node.js was still in its initial releases, it became extremely popular. Thousands of developers were using it. With the increasing usage, it became difficult for the Node.js core team to keep the software up to date with the user requirements. So thus formed the Node.js foundation in 2015.

Reviewer 2: ```suggestion
```

> In its earliest times, when Node.js was still in its initial releases, it became extremely popular. Thousands of developers were using it. With the increasing usage, it became difficult for the Node.js core team to keep the software up to date with the user requirements. So thus formed the Node.js foundation in 2015.

Reviewer: In its early* days,

> Node.js foundation is an open source community, and anyone can become a part of it by contributing to its projects. Open source doesn’t always have the best reputation for being friendly and welcoming, and that makes us sad. Everyone belongs in open source, and Node.js foundation is dedicated to making you feel welcome.

Reviewer: 1. After the first line, add a line, `Community outreach is an initiative under the umbrella of Node.js foundation to connect with the Node.js community.` 
2. In the second line, `Open source doesn't ...`, start it with, `We know, open source ...`.

> Node.js foundation is an open source community, and anyone can become a part of it by contributing to its projects. Community outreach is an initiative under the umbrella of Node.js foundation to connect with the Node.js community. We know, open source doesn’t always have the best reputation for being friendly and welcoming, and that makes us sad. Everyone belongs in open source, and Node.js foundation is dedicated to making you feel welcome.

Reviewer 2: ```suggestion
```

## Code of Conduct

In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, sex characteristics, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, religion, or sexual identity and orientation.

[Read the full code of conduct →](https://github.com/nodejs/admin/blob/master/CODE_OF_CONDUCT.md)

## Ways to Contribute

> We welcomes all kinds of contributions — documentation, bugs reporting, core contribution. Irrespective of the extent of the contribution, we encourage you to be a part of this initiative and own this Open Source project. If you miss a feature, request it, or maybe help to build it.

Reviewer: `Open Source` needs to be replaced with `open source`

> Node.js is an Open Source community and welcomes all kinds of contributions — documentation, bugs reporting, core contribution. Irrespective of the extent of the contribution, we encourage you to be a part of this initiative and own this Open Source project. If you miss a feature, request it, or maybe help to build it.

Reviewer: Need to change `Open Source` to open source.

Reviewer: `Node.js is an Open Source community` is repetitive; replace with `we welcome..`

- [Community Committee Meetings](https://github.com/nodejs/community-committee)

- [Core Contributions](https://github.com/nodejs/node)

- [Reporting a Bug](https://github.com/nodejs/node/issues)

- [Requesting a feature](https://github.com/nodejs/node/issues)

## Node.js News

### Stay in Touch

> Subscribe to the [Node.js official monthly mailing list](https://us14.campaign-archive.com/home/?u=c7c2e114a827812354112c23b&id=f006b61f29) to get all the latest update related to the NodeJS.

Reviewer: Change to Node.js

### Twitter

For updates about the Node.js, follow [@nodejs](https://twitter.com/nodejs)  on Twitter.

## Where to Get Support

### Node.js Help Repository

Node.js has a dedicated [repository on GitHub](https://github.com/nodejs/help) to help you with your issues. If you are facing a problem, feel free to create an issue, or go through the already answered questions to seek help.

### Stack Overflow

Mostly, community members prefer to ask questions at Stack Overflow. You can also ask questions there by using `node.js` tag or go through [already answered questions](https://stackoverflow.com/tags/node.js) for help.

### DEV Community

> DEV community is a great place to share and learn about Node.js. You can go through exisiting articles tagged with [#node](https://dev.to/t/node) to learn about Node.js or asked any question.

Reviewer: *ask any question.

# Content for NodeJS Community Webpage

## Introduction

NodeJS is an open source software built, used, and maintained by the community. We believe community to be at the core of NodeJS' popularity. Amazing folks from different parts of the world, with a common interest, collaborate and shape this community.

In its earliest times, when Node.js was still in its initial releases, it became extremely popular. Thousands of developers were using it. With the increasing usage, it became difficult for the Node.js core team to keep the software up to date with the user requirements. So thus formed the Node.js foundation in 2015.

Node.js foundation is an open source community, and anyone can become a part of it by contributing to its projects. Open source doesn’t always have the best reputation for being friendly and welcoming, and that makes us sad. Everyone belongs in open source, and Node.js foundation is dedicated to making you feel welcome.

## Code of Conduct

> Like every other community, the Node.js foundation has also defined some rules which every member needs to uphold. These rules are there to avoid any kind of misconduct and harassment.

Reviewer 2: I don't like the lingo here. It sounds childish and snappish. I think we can do a lot better here.

Reviewer 2: Add this:

```
In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, sex characteristics, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, religion, or sexual identity and orientation.

[Read the full code of conduct →](https://github.com/nodejs/admin/blob/master/CODE_OF_CONDUCT.md)

```

[Read the full code of conduct](https://github.com/nodejs/outreach/blob/master/CODE_OF_CONDUCT.md)

> ## Want to Contribute

Reviewer 2: ```suggestion
## Ways to contribute
```

> Node.js is an Open Source community, and all kinds of contributions — documentation, bugs reporting, core contribution — are welcomed. Irrespective of the size of the contribution, we encourage you to be a part of this initiative and own this Open Source project. If you miss a feature, request it or maybe help to build it.

Reviewer 2: Please run this through Grammarly. There are mistakes of formatting and grammar in there.

- [Comm Comm Meetings](https://github.com/nodejs/community-committee)

- [Core Contributions](https://github.com/nodejs/node)

- [Reporting a Bug](https://github.com/nodejs/node/issues)

- [Requesting a feature](https://github.com/nodejs/node/issues)

## Node.js news

### Stay in Touch

> Subscribe to the [NodeJS official monthly mailing list](https://us14.campaign-archive.com/home/?u=c7c2e114a827812354112c23b&id=f006b61f29) to get all the latest update related to the NodeJS.

Reviewer 2: It's `Node.js` and not `NodeJS` — we discussed that in the last meeting.

### Twitter

For the exciting news about the Node.js, [follow @nodejs on twitter](https://twitter.com/nodejs).

> ## Stuck at Somewhere

Reviewer 2: ```suggestion
## Where to get support
```

### Node.js Help Repository

> Because of the level of activity in the `nodejs/node` is so high, we created a separate repo for helping purpose. [Check it out now](https://github.com/nodejs/help/issues). There you'll find a lot of questions or requests for general help using Node.js. If you still can't find the solution to your problem, all you need to do is to open an issue. After that, someone from the Node.js foundation will contact you on that issue.

Reviewer 2: Too much information. A simple two line description would be enough.

### Stack Overflow

> Millions of developers are using Node.js on a regular basis. We have a huge community of developers on Stack Overflow where they have answered thousands of questions related to Node.js.

Reviewer 2: Again, check how it's done at https://www.gatsbyjs.org/contributing/community/

Less is more.

Read through the [existing questions](https://stackoverflow.com/tags/node.js) tagged with `node.js` or [ask your own](https://stackoverflow.com/questions/ask?tags=nodejs).

### Dev Community

Dev community is another great place to ask question related to Node.js. You can also read some informative articles there related Node.js. Hit here to read through the [existing question and posts](https://dev.to/t/node) tagged with node. You can also [ask for help](https://dev.to/new/help) or [write a post](https://dev.to/new/node) with the node tag.

> ## Learning Node

Reviewer 2: ```suggestion
## Learn More About Node.js:
```

The community has curated some free and high-quality resources for you to get started with Node.js.

> * Read Node.js [documentation](https://nodejs.org/en/docs/) to get started with the Node.js with a solid foundation.

Reviewer 2: No mention of Nodejs.dev — which is an excellent getting started resource.
